package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "wp-autoupdates",
```

Comment:
for completeness, shouldn't the name be something like "wordpress/wp-autoupdates"?  and shouldn't the version match the `WP_AUTO_UPDATES_VERSION` in `wp-autoupdates.php`?